src/components/Header/components/LocationSearch/LocationSearch.module.scss

```diff
@@ -0,0 +1,14 @@
+.currentLocation {
+  .title {
+    color: #aaaaaa;
+  }
+
+  .value {
+    display: inline-block;
```

Comment:
combination flex and block/inline-block, am I right?

Reply:
I was wrong. I have fixed it.